exporters/trace/build.gradle

```diff
@@ -1,7 +1,12 @@
+plugins {
+    id 'java'
```

Comment:
I think this can be moved to https://github.com/GoogleCloudPlatform/opentelemetry-operations-java/blob/5ecabaab3b959a8e882bd2333691137e49d35bc5/build.gradle#L2, so that we only make this change once to be used by all projects in the repo.

Reply:
Are you sure we need the `java` plugin?
Looks like `java-library` plugin itself supports `testImplementation`, it's mentioned on https://docs.gradle.org/current/userguide/java_library_plugin.html#sec:java_library_configurations_graph

Reply:
you're right. `gradle build` works without it.

Reply:
And `gradle test` as well?